src/hooks/useChangePasswordSchema.ts

```diff
@@ -0,0 +1,35 @@
+import { z } from 'zod';
```

Comment:
All these next hooks are just the same schema, but needed to be created as I am adding the `useTranslations` hook.